Lesson6/src/com/home/military/MilitaryUnit.java

```diff
@@ -74,6 +75,19 @@ private boolean doesRecruitAlreadyExists(Person person, Person[] recruits) throw
         return false;
     }
 
+    public void infoRecrutsMilitaryUnit(MilitaryUnit militaryUnit) {
+        String[] namesRecruits = new String[militaryUnit.recruitsArray.length];
```

Comment:
recruitsArray can contain null elements because memory is allocated by capacity. You need to check this